src/components/Dropdown/Dropdown.tsx

```diff
@@ -82,8 +84,10 @@ export const Dropdown = ({
       disabled={isDisabled}
       dropdownRender={dropdownRender}
       menu={menu}
+      open={open}
```

Comment:
Is the open prop really needed? Do you need to control visibility from the outside?

Reply:
Not as of now, but I thought it made sense to have it together with `onOpenChange`, plus its documentation refers to this prop. 
Do you think we should wait to add it until we actually need it?

Reply:
onOpenChange does not directly refers to the open prop, more on the current open state which is managed by the component itself. Open would allow you to control the dropdown visibility from the outside. While it makes sense to provide such feature I'd rather discourage its use tbh.

Reply:
Ok I get your point, I removed it and left only the `onOpenChange`, improving its documentation